pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -176,3 +180,17 @@ message TLSClientCertificateAuthenticationPolicy {
   // validate the remote TLS client.
   string client_certificate_authorities = 1;
 }
+
+message JWTAuthenticationPolicy {
+  // Algorithms and keys to use to authenticate the JWTs.
+  repeated JWTKeyData key = 1;
```

Comment:
Is there a need for actually supporting multiple keys here? There's already the AnyAuthenticationPolicy that you can use to instantiate multiple JWTAuthenticationPolicies and chain them.

Reply:
Just an artifact of me writing this PR without much knowledge of the rest of BuildBarn. Using `AnyAuthenticationPolicy` would allow for key rotation so I'm fine with removing the multiple key support in `JWTAuthenticationPolicy`.

Reply:
Also, if a key ID is included in the JWT header, then this could do a lookup from a map instead of iterating through the slice. Probably not worth implementing though since probably safe to assume there to only be one key in normal operation and two if keys are being rotated, then using a map won't really be a win over iterating through the one key (whether like this or with `AnyAuthenticationPolicy`).

Reply:
Exactly! It's most likely that people will only use a small number of keys.